.github/workflows/functional-networking.yaml

```diff
@@ -0,0 +1,59 @@
+name: functional-networking
+on:
+  pull_request:
+    paths:
+      - '^.*networking.*$'
+      - '.github/workflows/functional-networking.yaml'
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  functional-networking:
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["master"]
+        openstack_version: ["master"]
+        ubuntu_version: ["20.04"]
+        include:
+          - name: "xena"
+            openstack_version: "stable/xena"
+            ubuntu_version: "20.04"
+          - name: "wallaby"
+            openstack_version: "stable/wallaby"
+            ubuntu_version: "20.04"
+          - name: "victoria"
+            openstack_version: "stable/victoria"
+            ubuntu_version: "20.04"
+          - name: "ussuri"
+            openstack_version: "stable/ussuri"
+            ubuntu_version: "18.04"
+          - name: "train"
+            openstack_version: "stable/train"
+            ubuntu_version: "18.04"
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
+    name: Deploy OpenStack ${{ matrix.name }} with Neutron and run networking acceptance tests
+    steps:
+      - name: Checkout Gophercloud
+        uses: actions/checkout@v2
+      - name: Deploy devstack
+        uses: EmilienM/devstack-action@v0.6
+        with:
+          branch: ${{ matrix.openstack_version }}
+      - name: Checkout go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15'
+      - name: Run Gophercloud acceptance tests
+        run: ./script/acceptancetest
+        env:
+          DEVSTACK_PATH: ${{ github.workspace }}/devstack
+          ACCEPTANCE_TESTS_FILTER: "^.*networking.*$"
```

Comment:
Something's wrong, if you look at the jobs it's passing a different `ACCEPTANCE_TESTS_FILTER` env var to the acceptancetest, with a value of `^acceptance/openstack$`.

https://github.com/gophercloud/gophercloud/pull/2318/checks